locale/en-us/dialog/neon_confirmation.dialog

```diff
@@ -0,0 +1,2 @@
+Hello! I am Neon dot A.I., ready to assist.
```

Comment:
I think in other places (personal skill) it's just "Neon A.I." without the spoken "dot"